src/tests/test_performance/test_large_files.py

```
@@ -0,0 +1,553 @@
+                "audio": mock_audio_instance,
+                "video": mock_video_instance,
+                "gemini": mock_gemini_instance
+            }
+
+    @pytest.mark.slow
+    def test_large_audio_file_processing(self, large_audio_file, performance_settings, mock_processors_for_performance):
+        """Test processing of large audio files"""
+        start_time = time.time()
+        
+        # Create initial state
+        initial_state = create_initial_state(
```

Ruff (0.14.13)

Local variable `initial_state` is assigned to but never used

Remove assignment to unused variable `initial_state`

(F841)